src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  @include smooth;
```

Comment:
Ensure that the `smooth` mixin is defined in your SCSS files. If it's not, you need to define it to avoid compilation errors.